docs/workflow/building/libraries/cross-building.md

```diff
@@ -24,108 +23,102 @@ and for arm64 you need:
     $ sudo apt-get install binutils-aarch64-linux-gnu
 
 
-Generating the rootfs
+Generate the rootfs
 ---------------------
-The `eng/common/cross/build-rootfs.sh` script can be used to download the files needed for cross compilation. It will generate an Ubuntu 16.04 rootfs as this is what CoreFX targets.
+The `eng/common/cross/build-rootfs.sh` script can be used to download the files needed for cross compilation. It can generate rootfs for different operating systems.
 
-    Usage: ./eng/common/cross/build-rootfs.sh [BuildArch] [LinuxCodeName] [lldbx.y] [--skipunmount] --rootfsdir <directory>]
-    BuildArch can be: arm, armel, arm64, x86
-    LinuxCodeName - optional, Code name for Linux, can be: trusty, xenial(default), zesty, bionic, alpine. If BuildArch is armel, LinuxCodeName is jessie(default) or tizen.
-    lldbx.y - optional, LLDB version, can be: lldb3.9(default), lldb4.0, lldb5.0, lldb6.0 no-lldb. Ignored for alpine
+    Usage: ./eng/common/cross/build-rootfs.sh [BuildArch] [CodeName] [lldbx.y] [llvmx[.y]] [--skipunmount] --rootfsdir <directory>]
+    BuildArch can be: arm(default), arm64, armel, armv6, ppc64le, riscv64, s390x, x64, x86
+    CodeName - optional, Code name for Linux, can be: xenial(default), zesty, bionic, alpine, alpine3.13 or alpine3.14. If BuildArch is armel, LinuxCodeName is jessie(default) or tizen.
```

Comment:
This is a continuous maintenance. How do you feel if we just remove the usage from docs or point to `--help`? Most developers just use the prebuilt cross image from https://github.com/dotnet/versions/blob/main/build-info/docker/image-info.dotnet-dotnet-buildtools-prereqs-docker-main.json. Folks who do work with `build-rootfs.sh` directly can run `build-rootfs.sh --help` to find the usage.

Reply:
Removed this